treescript/src/treescript/data/treescript_task_schema.json

```diff
@@ -87,6 +90,16 @@
                         "required": ["path", "name", "platform_configs", "version_path"]
                     }
                 },
+                "merge_info": {
```

Comment:
I like this idea, although I'd want to add it to the other payload options as well; perhaps in the same PR as the 'valid action sets' so that we don't end up contradicting it.